test/e2e/delete_test.go

```diff
@@ -63,12 +125,20 @@ func testWindowsNodeDeletion(t *testing.T) {
 	// Phase is ignored during deletion, in this case we are just waiting for Machines to be deleted.
 	_, err = testCtx.waitForWindowsMachines(int(expectedNodeCount), "", true)
 	require.NoError(t, err, "Machine controller Windows machine deletion failed")
-	_, err = testCtx.waitForWindowsMachines(int(expectedNodeCount), "", false)
-	require.NoError(t, err, "ConfigMap controller Windows machine deletion failed")
+
+	// TODO: Currently on vSphere it is impossible to delete a Machine after its node has been deleted.
```

Comment:
I am obviously missing something. Where is the call that deletes the BYOH Machine?